editor/src/clj/editor/image.clj

```diff
@@ -37,9 +39,10 @@
         image ((:f content-generator) (:args content-generator))]
     (g/precluding-errors
       [image]
-      (let [texture-image (tex-gen/make-texture-image image texture-profile compress?)]
+      (let [texture-generator-result (tex-gen/make-texture-image image texture-profile compress?)
+            texture-protobuf-bytes (TextureUtil/generateResultToByteArray texture-generator-result)]
```

Comment:
It's not really a `texture-protobuf` anymore, right? Maybe `texture-resource-bytes` or `texturec-bytes`?